.github/workflows/deploy.yml

```diff
@@ -119,7 +119,7 @@ jobs:
           export TF_VAR_paas_app_docker_image=${{ env.DOCKERHUB_REPOSITORY_BASE }}${{ env.DOCKERHUB_ENV }}:${{ env.TAG }}
           cd terraform/app
           terraform init -reconfigure -input=false -backend-config="bucket=paas-s3-broker-prod-lon-7f2ca242-9929-4662-a79c-c454ea56ea7b"
-          terraform apply -input=false -auto-approve -var-file ../workspace-variables/dev.tfvars -var='secret_paas_app_env_values={"RAILS_MASTER_KEY":"${{secrets.RAILS_MASTER_KEY_DEV}}"}'
+          terraform apply -input=false -auto-approve -var-file ../workspace-variables/dev.tfvars -var='secret_paas_app_env_values={"RAILS_MASTER_KEY":"${{secrets.RAILS_MASTER_KEY_DEV}}"}' -var='environment=${{github.event.inputs.environment}}' -var='secret_paas_app_env_values={"SYSLOG_DRAIN_URL":"${{secrets.SYSLOG_DRAIN_URL}}"}'
```

Comment:
I think we already have `secret_paas_app_env_values` field that we pass to terraform - no need to add another one, just use the existing one (with RAILS_MASTER_KEY).

With this, you get a failed deployment with 
```
Error: Reference to undeclared input variable

  on terraform.tf line 47, in module "paas":
  47:   logstash_url                      = var.SYSLOG_DRAIN_URL

An input variable with the name "SYSLOG_DRAIN_URL" has not been declared. This
variable can be declared with a variable "SYSLOG_DRAIN_URL" {} block.
```

I imagine because when terraform gets two variables with the same name in options (like you do here) it takes the first one?

Reply:
@kicferk1 where do you see that error log? I couldn't see it on Github

Reply:
I got an email pointing me to https://github.com/DFE-Digital/early-careers-framework/actions/runs/663581993